sys-apps/fwupd/files/fwupd-0.9.2-meson-dist.patch

```diff
@@ -0,0 +1,41 @@
+--- fwupd-0.9.2/meson.build	2017-05-22 13:37:19.000000000 -0500
++++ fwupd-0.9.2/meson.build	2017-06-17 02:37:45.195035820 -0500
+@@ -203,21 +203,23 @@ subdir('po')
+ subdir('policy')
+ subdir('src')
+ 
+-archiver = find_program('git', required : false)
+-if archiver.found()
+-  run_target('dist',
+-    # git config tar.tar.xz.command "xz -c"
+-    command: [
+-      'git', 'archive',
+-      '--prefix=' + meson.project_name() + '-' + meson.project_version() + '/',
+-      'HEAD',
+-      '--format=tar.xz',
+-      '--output',
+-      meson.project_name() + '-' + meson.project_version() + '.tar.xz'
+-    ]
+-  )
+-else
+-  message('git not found, you will not be able to run `ninja dist`')
++if meson.version().version_compare('<0.41.0')
++  archiver = find_program('git', required : false)
++  if archiver.found()
++    run_target('dist',
++      # git config tar.tar.xz.command "xz -c"
++      command: [
++        'git', 'archive',
++        '--prefix=' + meson.project_name() + '-' + meson.project_version() + '/',
++        'HEAD',
++        '--format=tar.xz',
++        '--output',
++        meson.project_name() + '-' + meson.project_version() + '.tar.xz'
++      ]
++    )
++  else
++    message('git not found, you will not be able to run `ninja dist`')
++  endif
+ endif
+ 
+ meson.add_install_script('meson_post_install.sh', systemd.get_pkgconfig_variable('systemdsystemunitdir'), localstatedir)
```

Comment:
There, that should fix it up. Question, is there a tool used for that sort of thing, or is it just something you need to do manually?